examples/tensorrt_llm/configs/llmapi_disagg_configs/single_node_config.yaml

```diff
@@ -48,4 +48,4 @@ generation_servers:
     enable_overlap_scheduler: true
     use_cuda_graph: false
   urls:
-      - "localhost:8002"
```

Comment:
can you unstage this file's changes so it's clearer that these files are actually unchanged since an older commit?